api.go

```diff
@@ -316,6 +316,13 @@ type SearchOptions struct {
 	// Trim the number of results after collating and sorting the
 	// results
 	MaxDocDisplayCount int
+
```

Comment:
The struct fields in the `SearchOptions` struct, which are passed through the RPC boundary. To see how they are used by the RPC client (`frontend`), go to the sister PR.

[Next](https://github.com/sourcegraph/sourcegraph/pull/12548/files#diff-39413ee2a9c32387903e199473779d74R200)